MetaMask/metamask-extension
# fix: update Fuse logo

## **Description**

This PR updates the Fuse mainnet logo. The motivation behind this change is to ensure that the Fuse network's branding is accurately represented in MetaMask, providing users with a more reliable and visually consistent experience across the platform. The previous logo used in the wallet was outdated, and this update brings the visual in line with the current branding guidelines of the Fuse network. This improvement enhances the user interface by aligning the network's visual representation with its most current branding, ensuring that users can easily recognize and associate the Fuse mainnet within MetaMask.

[![Open in GitHub Codespaces](https://github.com/codespaces/badge.svg)](https://codespaces.new/MetaMask/metamask-extension/pull/PR?quickstart=1)

## **Related issues**

Fixes: 

## **Manual testing steps**

1. Navigate to the MetaMask extension and select the network dropdown.
2. Choose the Fuse mainnet from the list of available networks.
3. Observe the updated Fuse logo, confirming it matches the current branding guidelines provided by the Fuse network.

## **Screenshots/Recordings**



### **Before**

![Screenshot 2024-02-28 at 11 12 47](https://github.com/MetaMask/metamask-extension/assets/30021476/63d66074-63b5-45ec-8e9d-10e404007e38)

### **After**



## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
